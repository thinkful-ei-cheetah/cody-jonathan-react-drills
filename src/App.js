import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';
import Accordian from './state-drills/Accordian';

const sections = [
  {
    title: 'Section 1',
    content: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
  },
  {
    title: 'Section 2',
    content: 'Cupiditate tenetur aliquam necessitatibus id distinctio quas nihil ipsam nisi modi!',
  },
  {
    title: 'Section 3',
    content: 'Animi amet cumque sint cupiditate officia ab voluptatibus libero optio et?',
  },
]


class App extends Component {
  render() {
    return (
      <div>
        <Accordian sections={sections} />
      </div>
    );
  }
}

export default App;
