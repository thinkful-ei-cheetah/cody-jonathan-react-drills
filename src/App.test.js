import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import App from './App'
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() })
describe(`Tabs Component`, () => {
  const tabsProp = [/* contents not shown */]

  it('renders without errors', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  })

  it('renders empty given no App', () => {
    const wrapper = shallow(<App />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the first tab by default', () => {
    const wrapper = shallow(<App tabs={tabsProp} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
