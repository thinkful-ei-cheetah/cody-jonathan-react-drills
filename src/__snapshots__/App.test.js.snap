// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stage Participants renders UI as expected 1`] = `
<div>
  <ul>
    <li>
      <button
        onClick={[Function]}
      >
        Section 1
      </button>
      <p />
    </li>
    <li>
      <button
        onClick={[Function]}
      >
        Section 2
      </button>
      <p />
    </li>
    <li>
      <button
        onClick={[Function]}
      >
        Section 3
      </button>
      <p />
    </li>
  </ul>
</div>
`;

exports[`Tabs Component renders empty given no App 1`] = `
<div>
  <Accordian
    sections={
      Array [
        Object {
          "content": "Lorem ipsum dolor sit amet consectetur adipisicing elit.",
          "title": "Section 1",
        },
        Object {
          "content": "Cupiditate tenetur aliquam necessitatibus id distinctio quas nihil ipsam nisi modi!",
          "title": "Section 2",
        },
        Object {
          "content": "Animi amet cumque sint cupiditate officia ab voluptatibus libero optio et?",
          "title": "Section 3",
        },
      ]
    }
  />
</div>
`;

exports[`Tabs Component renders the first tab by default 1`] = `
<div>
  <Accordian
    sections={
      Array [
        Object {
          "content": "Lorem ipsum dolor sit amet consectetur adipisicing elit.",
          "title": "Section 1",
        },
        Object {
          "content": "Cupiditate tenetur aliquam necessitatibus id distinctio quas nihil ipsam nisi modi!",
          "title": "Section 2",
        },
        Object {
          "content": "Animi amet cumque sint cupiditate officia ab voluptatibus libero optio et?",
          "title": "Section 3",
        },
      ]
    }
  />
</div>
`;
